src/applications/vaos/redux/selectors.js

```diff
@@ -36,17 +36,12 @@ export const selectFeatureExpressCare = state =>
 export const selectFeatureExpressCareNewRequest = state =>
   toggleValues(state).vaOnlineSchedulingExpressCareNew;
 export const selectFeatureToggleLoading = state => toggleValues(state).loading;
-const selectFeatureFlatFacilityPage = state =>
-  toggleValues(state).vaOnlineSchedulingFlatFacilityPage;
-const selectFeatureFlatFacilityPageSacramento = state =>
-  toggleValues(state).vaOnlineSchedulingFlatFacilityPageSacramento;
+// const selectFeatureFlatFacilityPage = true;
```

Comment:
You can go ahead and remove these comments. Also, would be good to remove the entries for these toggles in the featureFlagNames.js file.

Reply:
removed comments and toggle names from featureFlagNames.js